<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSrvDlg</class>
 <widget class="QDialog" name="FileSrvDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>文件发送</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LIneEdit1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Devanagari</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="sfileNameLineEdit_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Devanagari</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>进度</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="sendProgressBar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="srvClosePushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>138</x>
     <y>101</y>
     <width>279</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="sfileNameLineEdit">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="openFilePushButton">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="sendFilePushButton">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>371</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="sfileNameLineEdit_2">
    <property name="font">
     <font>
      <family>Adobe Devanagari</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>大小:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sfileSizeLineEdit">
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="text">
     <string>已发送：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendSizeLineEdit">
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
